orcestra_book/plans/HALO-20240827a.md

```diff
@@ -0,0 +1,206 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.12
+    jupytext_version: 1.7.1
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+platform: HALO
+flight_id: HALO-20240827a
+takeoff: "2024-08-22 11:10:00Z"
+landing: "2020-08-22 20:10:00Z"
```

Comment:
The dates (and year) are not updated

Reply:
thanks, I changed that in the latest update